spec/features/accessibility/work_packages/work_package_query_spec.rb

```diff
@@ -40,8 +40,8 @@
   def visit_index_page
     work_packages_page.visit_index
     # ensure the page is loaded before expecting anything
-    find('.advanced-filters--filters select option', text: /\AAssignee\Z/,
-                                                     visible: false)
+    expect(page).to have_selector('#operators-status_id', visible: false),
+                    "Page was not fully loaded"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.